Makefile

```diff
@@ -20,12 +21,20 @@ HOST_TARGETS := $(INSTALL_WIN_TARGET) $(INSTALL_UNIX_TARGET) $(HOST_APP_TARGET)
 
 all: $(HOST_TARGETS)
 
+install: install-unix
+install-unix: $(HOST_TARGETS)
+	$(INSTALL_UNIX_TARGET) --local $(BROWSER)
+install-win: $(HOST_TARGETS)
+	$(INSTALL_UNIX_TARGET) --local $(BROWSER)
+
 %/.d:
 	mkdir -p $(@D)
 	@touch $@
 
 $(HOST_TARGETS): $(TARGET_DIR)/%: $(SRC_DIR)/% $(TARGET_DIR)/.d
 	sed -e "s/_VERSIONHOLDER_/$(VERSION)/g" < $(SRC_DIR)/$* > $@
+	# Make install script executable
+	echo $@ | grep -vq '.sh$$' || chmod a+x $@
```

Comment:
Your suggestion is much nicer, but it also breaks the logic: I would expect the “build” target to leave me in a fully functional state, where I can go and run the script without having to change its permission.

Reply:
We could also run it with `sh install_host_app.sh` without setting it executable.

Should `passff.py` also be executable like its source version in `src/`?

Reply:
We could, but it's an extra burden on the user, where really this should be handled by the shebang in an executable script. I'm happy to change if you insist, but I would otherwise keep the ugly hidden in the Makefile.

Reply:
Yeah, the Python script should also be executable, I think.

Reply:
Let's keep the ugly hidden in the Makefile :+1: